### PR TITLE
Unit Testing for HybridContentManager rename

### PR DESCRIPTION
**Pull request by mitchellsayer**
Relates to https://github.com/viaduct-ai/backlog/issues/242

This pr removes some code from utils so that unit tests will compile correctly, transitioned to pipenv, and implemented unit tests for the `rename` function implemented by devin in pr #3.

### CODE REVIEW

File: pgcontents/tests/test_hybrid_manager.py [hunk 1]
```diff
@@ -291,18 +260,59 @@ def test_cant_delete_root(self):
...
 
         with assertRaisesHTTPError(self, 400):
-            cm.rename('Untitled.ipynb', 'A/Untitled.ipynb')
+            cm.rename('', 'A')
+
+    def test_rename_submanager_calls(self):
+        cm = self.contents_manager
+        cm.new_untitled(ext='.ipynb')
+
+        old_path = 'Untitled.ipynb'
```
Comment (devstein) [inline pgcontents/tests/test_hybrid_manager.py]: Make `Untitled.ipynb` a variable

File: pgcontents/tests/test_hybrid_manager.py [hunk 2]
```diff
@@ -291,18 +260,59 @@ def test_cant_delete_root(self):
...
+        # Run tests
+        old_manager.delete.assert_called_with(old_path)
+        old_manager.get.assert_called_with(old_path)
+        new_manager.save.assert_called_with(old_model, new_relative_path)
+
+    def test_can_rename_across_managers(self):
+        cm = self.contents_manager
+        cm.new_untitled(ext='.ipynb')
+
+        old_path = 'Untitled.ipynb'
```
Comment (devstein) [inline pgcontents/tests/test_hybrid_manager.py]: Same

File: pgcontents/tests/test_hybrid_manager.py (hunk 1 above)
Comment (mitchellsayer) [reply pgcontents/tests/test_hybrid_manager.py]: do you mean make it a global variable?

Comment (devstein) [reply pgcontents/tests/test_hybrid_manager.py]: yes

**Pull request merged**